docs/open-source/index.mdx

```diff
@@ -1,8 +1,18 @@
 ---
 title: Open source
-description: TODO - filler text so that the SEO check doesn't complain!
+description: Qiskit open-source introduction
 ---
 
 # Open source
 
-Blah blah
+Since 2016, IBM Quantum has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.
```

Comment:
```suggestion
Since 2016, IBM Quantum&trade; has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.
```